internal/mocks/mockclient/mocks.go

```diff
@@ -0,0 +1,5 @@
+package mockclient
```

Comment:
Haha, I literally ended up having to do the same thing. Feel free to combine the comments after I merge mine in.